netmanager-app/components/Analytics/PollutantCategory.tsx

```
@@ -0,0 +1,101 @@
+          </Avatar>
+        </CardContent>
+      </Card>
+      {show && (
+        <ul className="absolute z-10 w-full mt-2 bg-white border rounded-md shadow-lg max-h-60 overflow-auto">
+          {sortedData.length === 0 ? (
+            <li className="p-2 text-gray-500">No data</li>
+          ) : (
+            sortedData.map((data, index) => (
+              <li key={index} className="p-2 hover:bg-gray-100">
+                {data.label} - <span className={`font-semibold text-${getPm25LevelColor(pm25level)}`}>
+                  {data.pm2_5 && !isNaN(data.pm2_5) ? data.pm2_5.toFixed(2) : 'N/A'}
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)